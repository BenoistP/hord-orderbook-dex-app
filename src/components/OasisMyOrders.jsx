import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';
// import ImmutablePropTypes from 'react-immutable-proptypes';

import './OasisMyOrders.scss';


const propTypes = PropTypes && {
};
const defaultProps = {};


class OasisMyOrders extends PureComponent {
  render() {
    return (
      <div className="OasisMyOrders">
        OasisMyOrders
      </div>
    );
  }
}

OasisMyOrders.displayName = 'OasisMyOrders';
OasisMyOrders.propTypes = propTypes;
OasisMyOrders.defaultProps = defaultProps;
export default OasisMyOrders;
