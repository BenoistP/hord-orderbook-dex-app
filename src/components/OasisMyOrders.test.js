/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import { shallow } from 'enzyme';
import OasisMyOrders from './OasisMyOrders';

describe('(Component) OasisMyOrders', () => {
  it('should render', () => {
    const props = {};
    const wrapper = shallow(
      <OasisMyOrders {...props}/>,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
