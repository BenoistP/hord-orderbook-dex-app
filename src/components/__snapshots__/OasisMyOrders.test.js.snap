// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) OasisMyOrders should render 1`] = `
<div
  className="OasisMyOrders"
>
  OasisMyOrders
</div>
`;
